apis/sfn/v1alpha1/zz_enums.go

```diff
@@ -0,0 +1,112 @@
+	HistoryEventType_MapStateExited               HistoryEventType = "MapStateExited"
+	HistoryEventType_MapStateFailed               HistoryEventType = "MapStateFailed"
+	HistoryEventType_MapStateStarted              HistoryEventType = "MapStateStarted"
+	HistoryEventType_MapStateSucceeded            HistoryEventType = "MapStateSucceeded"
+	HistoryEventType_ParallelStateAborted         HistoryEventType = "ParallelStateAborted"
+	HistoryEventType_ParallelStateEntered         HistoryEventType = "ParallelStateEntered"
+	HistoryEventType_ParallelStateExited          HistoryEventType = "ParallelStateExited"
+	HistoryEventType_ParallelStateFailed          HistoryEventType = "ParallelStateFailed"
+	HistoryEventType_ParallelStateStarted         HistoryEventType = "ParallelStateStarted"
+	HistoryEventType_ParallelStateSucceeded       HistoryEventType = "ParallelStateSucceeded"
+	HistoryEventType_PassStateEntered             HistoryEventType = "PassStateEntered"
+	HistoryEventType_PassStateExited              HistoryEventType = "PassStateExited"
+	HistoryEventType_SucceedStateEntered          HistoryEventType = "SucceedStateEntered"
+	HistoryEventType_SucceedStateExited           HistoryEventType = "SucceedStateExited"
+	HistoryEventType_TaskFailed                   HistoryEventType = "TaskFailed"
+	HistoryEventType_TaskScheduled                HistoryEventType = "TaskScheduled"
+	HistoryEventType_TaskStarted                  HistoryEventType = "TaskStarted"
+	HistoryEventType_TaskStartFailed              HistoryEventType = "TaskStartFailed"
+	HistoryEventType_TaskStateAborted             HistoryEventType = "TaskStateAborted"
+	HistoryEventType_TaskStateEntered             HistoryEventType = "TaskStateEntered"
+	HistoryEventType_TaskStateExited              HistoryEventType = "TaskStateExited"
+	HistoryEventType_TaskSubmitFailed             HistoryEventType = "TaskSubmitFailed"
+	HistoryEventType_TaskSubmitted                HistoryEventType = "TaskSubmitted"
+	HistoryEventType_TaskSucceeded                HistoryEventType = "TaskSucceeded"
+	HistoryEventType_TaskTimedOut                 HistoryEventType = "TaskTimedOut"
+	HistoryEventType_WaitStateAborted             HistoryEventType = "WaitStateAborted"
+	HistoryEventType_WaitStateEntered             HistoryEventType = "WaitStateEntered"
+	HistoryEventType_WaitStateExited              HistoryEventType = "WaitStateExited"
+)
+
+type LogLevel string
+
+const (
+	LogLevel_ALL   LogLevel = "ALL"
+	LogLevel_ERROR LogLevel = "ERROR"
+	LogLevel_FATAL LogLevel = "FATAL"
+	LogLevel_OFF   LogLevel = "OFF"
+)
+
+type StateMachineStatus_SDK string
+
+const (
+	StateMachineStatus_SDK_ACTIVE   StateMachineStatus_SDK = "ACTIVE"
+	StateMachineStatus_SDK_DELETING StateMachineStatus_SDK = "DELETING"
+)
+
+type StateMachineType string
+
+const (
+	StateMachineType_STANDARD StateMachineType = "STANDARD"
+	StateMachineType_EXPRESS  StateMachineType = "EXPRESS"
+)
```

Comment:
Where are these enums being used/referenced?

Reply:
Some of them are used and some of them aren't. Technically we can skip generating those but they are available in the pipeline and I though they could be useful for people who import these as packages to know what values are available programmatically.

Reply:
Makes sense